packages/core/core/src/BundleGraph.js

```diff
@@ -233,6 +233,10 @@ export default class BundleGraph {
     }
   }
 
+  createBundleReference(from: Bundle, to: Bundle): void {
+    this._graph.addEdge(from.id, to.id, 'references');
```

Comment:
@padmaia this probably conflicts with the current use of reference edges between bundles. Where is that information used? We should probably have different edge names as these seem orthogonal.